include/oneapi/tbb/parallel_for.h

```diff
@@ -179,6 +179,33 @@ task* start_for<Range, Body, Partitioner>::cancel(execution_data& ed) {
     return nullptr;
 }
 
+template<typename BasePartitioner>
+template<typename Range, typename Body>
+void numa_partitioner<BasePartitioner>::execute_for(const Range& range, const Body& body) const{
```

Comment:
Is it necessary to define a member function of numa_partitioner inside of the parallel_for header. That seems very unusual.